site-permissions/site-permissions-store/src/main/java/com/duckduckgo/site/permissions/store/SitePermissionsPreferences.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.site.permissions.store
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.SharedPreferences
+import javax.inject.Inject
+
+interface SitePermissionsPreferences {
+
+    var askCameraEnabled: Boolean
+    var askMicEnabled: Boolean
+}
+
+class SitePermissionsPreferencesImp @Inject constructor(private val context: Context) : SitePermissionsPreferences {
+
+    private val preferences: SharedPreferences
+        get() = context.getSharedPreferences(FILENAME, Context.MODE_PRIVATE)
+
+    override var askCameraEnabled: Boolean
```

Comment:
when will be this false? I'm trying to see that scenario but I can't find it.

Reply:
When site permissions settings is implemented (next PR) users can disable it. In this PR was added to define logic around showing site permissions dialog, however, as you mentioned, it will be always true in this PR